mage_ai/data_preparation/repo_manager.py

```diff
@@ -120,3 +130,51 @@ def set_repo_path(repo_path: str) -> None:
 
 def get_variables_dir(repo_path: str = None) -> str:
     return get_repo_config(repo_path=repo_path).variables_dir
+
+
+def create_secret(name: str, value: str):
```

Comment:
Move the secrets related methods into a separate file. It doesn't seem that they should be in this repo_manager.py.